src/systemtap/sssd_functions.stp

```diff
@@ -64,3 +64,61 @@ function sssd_acct_req_probestr(fc_name, entry_type, filter_type,
                        filter_value, extra_value)
     return probestr
 }
+
+function dp_target_str(target)
+{
+    if (target == 0) {
+        str_target = "ID"
+    } else if (target == 1) {
+        str_target = "AUTH"
+    } else if (target == 2) {
+        str_target = "ACCESS"
+    } else if (target == 3) {
+        str_target = "CHPASS"
+    } else if (target == 4) {
+        str_target = "SUDO"
+    } else if (target == 5) {
+        str_target = "AUTOFS"
+    } else if (target == 6) {
+        str_target = "SELINUX"
+    } else if (target == 7) {
+        str_target = "HOSTID"
+    } else if (target == 8) {
+        str_target = "SUBDOMAINS"
+    } else if (target == 9) {
+        str_target = "TARGET_SENTINEL"
```

Comment:
I would prefer to define "constants" in the beginning of file.

`/usr/share/systemtap/examples/process/schedtimes.stp` has something like this:
```
//constants
global DEAD=-1, RUNNING=1, QUEUED=2, SLEEPING=3
```

I am not sure whether these constant will be reused but one never know :-)